src/server/lib/middleware/index.ts

```diff
@@ -32,7 +32,7 @@ export const applyMiddleware = (server: Express): void => {
 
 	server.use(loggerMiddleware);
 	server.use(csrfMiddleware as RequestHandler);
-	// eslint-disable-next-line @typescript-eslint/no-misused-promises -- express has its own way of handling async middleware
+
```

Comment:
I have no idea when I made this change 😖  ... still it seems like the right thing to do as eslint does not complain about the following or any other line